src/libs/auth.js

```diff
@@ -1,12 +1,12 @@
-import { parseJSON } from 'date-fns/fp'
+import { addDays, parseJSON } from 'date-fns/fp'
 import _ from 'lodash/fp'
-import { Fragment } from 'react'
```

Comment:
We need to keep `Fragment`. It's still used on line 230.

Reply:
Yeah not sure how/why that got removed haha